Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -30,6 +30,15 @@ bool BreakPoints::IsTempBreakPoint(u32 address) const
   });
 }
 
+const TBreakPoint* BreakPoints::GetBreakPointAt(u32 address) const
```

Comment:
Is there a non-const version of it or a way to modify the breakpoint (if it's a logging one or not, if it's enabled, etc.)?

Reply:
Not yet! :(
There isn't a way to edit existing breakpoints afaict - beyond doing a saving to text, then clearing, editing the text, then loading. I have a few changes to breakpoints tentatively planned (including allowing changes to existing breakpoints), and at that point I'll need a non-const version, but I didn't want to add it yet, unused.